front-end/src/components/header/Header.scss

```
@@ -1,24 +1,35 @@
 @import "../../scss/variables";
 
 .header {
-  background: $black;
-  display: flex;
-  justify-content: flex-end;
-  color: $secondary-text-color;
+  background: $white;
+  color: $black;
+  padding: 16px 0;
+  &Box {
+    align-items: center;
+    display: flex;
+  }
 }
 
-.title {
-  font-size: $font-size;
-  margin: 15px 0; 
+.Search {
```

```suggestion
.search {
```
We need to be consistent with naming